### DISCUSSION TITLE
M2O & M2M relationship - Data viewing based on the specific type of information

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#11860] [Feature Requests]
Is there a way we can display or assign items from collection2 based on relevant field Client_Id’s? 

**Example** 
Collectiion1_Client_Id[eq]=Collection2_Client_Id

![image](https://user-images.githubusercontent.com/36895131/155759374-328d1a08-ba29-4106-97fd-932697f8ff3f.png)

**We tried configuring it but UnGroupedCollection has the items related to this Client_Id but it is not showing**

![image](https://user-images.githubusercontent.com/36895131/155759429-cdafb29f-e9ad-4f21-8f03-ef60a2a54480.png)

**Without Filter UnGroupedCollection all the items are showing**

![image](https://user-images.githubusercontent.com/36895131/155759468-275e6785-702e-4ab6-ba37-5030b35c5681.png)

**Comment by rijkvanzanten** [comment]
There's currently no way to use the value of the parent item in the filter of the relational interface. There's an active discussion around that here: https://github.com/directus/directus/discussions/2993

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
